### ISSUE TITLE
[ANDROID]steps to check after cancellation

### ISSUE THREAD

**Issue by no-toodate** [j3k0/cordova-plugin-purchase#1276]
Hello!

I would like to know, if a user cancels and interrupts the paiment process, what step should be done. 
I read the docs, but  I don't know if I have to refresh the store/product. 

Thank you for your help! 
### system info
```
Windows 10
Cordova 10.0.0
Device: Android 10
Plugin Version: 10.3
```

### Expected behavior
Be able to choose another product and pay for it directly afterwards.

### Observed behavior
Once a cancellation is done, and another product is chosen, nothing happens.

**Comment by j3k0** [comment]
Can you share some logs, in principle it should work as expected. Is it possible there is an exception thrown by some handlers?

**Comment by no-toodate** [comment]
Hello, 
I am so sorry about the lack orf response. I had been called on other features on my app so I had to put this matter in the backburner.
I'll try to get some logs. 
But from what I just tried, by accident. I accidentaly closed th pop up (generated by the order() function). 

now, when I try to order a product, the status is cancelled still. A small bit of code to show

 ```
 this.store.when(this.valeurEnvoi.app).cancelled((order) => {
            alert("annule");
           // what in the world am I supposed to do there ? 
        });
```
I automatically have 'annule' as soon as I tap the buy button. I tried order.finish(), refresh()... to no avail... 

I am redoing this from the base, to have something clean, but for now, I am stuck on this. I'll give you logs as soon as I can.
